test/pytest/receiver/test_receiver.py

```diff
@@ -0,0 +1,151 @@
+import logging
+from pathlib import Path
+import sys
+import time
+import pytest
+
+receiver_path = (
+    Path(__file__).parent.parent.parent.parent / "src/hidra/receiver")
```

Comment:
isn't there a nicer way to write this? :-)

Reply:
Unfortunately, I didn't find a better easy way. A real solution is to make Hidra "pip installable" which would likely require major restructuring of the file hierarchy and the packaging scripts. This is on my list.

Reply:
that may be not a real priority. pip is imo inferior to most other installation methods; so it might not be worth the hassle

Reply:
If a project is "pip installable", modules can be imported normally and IDEs will be able to navigate the code, show docstrings and autocompletions. 

I strongly recommend to always make a project "pip installable" from the beginning to avoid the problems the current Hidra setup has, e.g., the need for ugly lines like the above.

Reply:
I'd argue that it is a byproduct. most projects are pip-installable from the beginning; but in itself is not a good requirement.
beyond that -- despite the "easy usability" and "user friendly" promises of Python I find the whole navigation and file system interaction of Python extremely irritating and annoying. I have to yet come across a project that does this in a consistent, easy-to-read way